.eslintrc.yml

```diff
@@ -0,0 +1,18 @@
+env:
```

Comment:
why yml? =)

Reply:
it's easier to read, and easily converted to JSON if needed